test/controllers/project_skill_controller_test.exs

```diff
@@ -1,42 +1,25 @@
 defmodule CodeCorps.ProjectSkillControllerTest do
-  use CodeCorps.ApiCase
-
-  alias CodeCorps.ProjectSkill
-  alias CodeCorps.Repo
-
-  defp build_payload, do: %{ "data" => %{"type" => "project-skill", "attributes" => %{}}}
+  use CodeCorps.ApiCase, resource_name: :project_skill
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
-      conn = get conn, project_skill_path(conn, :index)
-      assert json_response(conn, 200)["data"] == []
+    [project_skill_1, project_skill_2] = insert_pair(:project_skill)
+
+     conn
+     |> request_index
+     |> json_response(200)
+     |> assert_ids_from_response([project_skill_1.id, project_skill_2.id])
     end
 
     test "filters resources on index", %{conn: conn} do
-      elixir = insert(:skill, title: "Elixir")
-      phoenix = insert(:skill, title: "Phoenix")
-      rails = insert(:skill, title: "Rails")
+      [project_skill_1, project_skill_2] = insert_pair(:project_skill)
```

Comment:
You figured out the missing third record, so I'll just link https://github.com/code-corps/code-corps-api/pull/425/files#r86388037 as a pointer on how to write this in one line.